src/set_relation/set_relation_test.cc

```diff
@@ -1,5 +1,4 @@
-/*!
- * \file set_relation_test.cc
+ /* file set_relation_test.cc
```

Comment:
No need to take out the \ in \file.  I think that is used by doxygen when creating docs for the code.

Reply:
Okay